caffe2/python/ideep/convnet_benchmarks.py

```diff
@@ -697,7 +697,7 @@ def GetArgumentParser():
         help="If True, dump the model prototxts to disk."
     )
     parser.add_argument("--net_type", type=str, default="simple")
-    parser.add_argument("--num_workers", type=int, default=1)
+    parser.add_argument("--num_workers", type=int, default=2)
```

Comment:
This change looks irrelevant to the rest

Reply:
I was testing the stuff and set it to `1` and checked it into accidentally. Reverted it back to `2`.